packages/firestore/src/remote/backoff.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google LLC
+ * Copyright 2020 Google LLC
```

Comment:
If you change this back to 2017 and revert the import reordering, this file could now be unchanged in this PR.

Reply:
Done